rpc_jobs/rpc_openstack.yml

```diff
@@ -234,6 +234,32 @@
     jobs:
       - 'PM_{repo_name}-{branch}-{image}-{scenario}-{action}'
 
+- project:
+    name: "rpc-openstack-master-rc-mnaio-postmerge"
+    repo_name: "rpc-openstack"
+    repo_url: "https://github.com/rcbops/rpc-openstack"
+    branch: "master-rc"
+    jira_project_key: "RO"
+    image:
+         - xenial_mnaio_no_artifacts:
+             FLAVOR: "onmetal-io2"
+             IMAGE: "OnMetal - Ubuntu 16.04 LTS (Xenial Xerus)"
+             REGIONS: "IAD"
+             FALLBACK_REGIONS: "DFW"
+       scenario:
```

Comment:
Hey @zreichert , the indentation here looks a bit funky.

Reply:
Yeah thats not right.  I'll push a fix.

Reply:
@mattt416 Have a look now